### DISCUSSION TITLE
MeteorRN Showcase

### DISCUSSION THREAD

**Discussion by not-ai-yet** [TheRealNate/react-native-meteor#51] [General]
Hi everyone,

If you are using Meteor React Native in a production app, we'd love to hear about it. Let us know on this thread.

**Comment by mvogttech** [comment]
[Whazzup](https://whazzup.co/) uses Meteor React Native!

**Reply by not-ai-yet** [reply]
Hey @mvogttech, thanks for letting me know! I've added your app to the showcase on the repo's README. If you have a short snippet you want to be displayed with your app's name and logo just let me know.
